explainer/fenced_frames_with_local_unpartitioned_data_access.md

```diff
@@ -139,25 +139,149 @@ Certain privacy preserving aggregated reports can be allowed from the fenced fra
 
 ## Click listener API
 
-The exact surface of the API is still being designed, but at a high level, there will be 2 parts to this API surface: 
+The click listener API is broken into two parts:
 
+*   The embedding context will invoke `addEventListener()` on the `HTMLFencedFrameElement` to listen for a click event on the fenced frame.
+*   A new method on `window.fence` that a script inside the fenced frame tree can invoke, which will trigger the embedding context’s click event listener.
 
+### Changes to HTMLFencedFrameElement
 
-*   The embedding context will invoke an API on the fenced frame element to listen for a click event on the fenced frame.
-*   An API that a script inside the fenced frame tree can invoke which will trigger the embedding context’s click event listener.
+After a fenced frame element object is created, the embedder can call `addEventListener(‘fencedtreeclick’, callback)` on it to attach an event listener to the frame. The new listener can fire when a event with type `click` is fired in the embedded document’s DOM tree. In addition, the `onfencedtreeclick` property will be added to fenced frame elements to mirror other event listeners with this syntax. 
```

Comment:
Done.

Reply:
Note that almost every event handler content attribute applies to _all_ HTML elements, with only a few belonging to  specifically `<body>`. See https://html.spec.whatwg.org/C#attributes-3:event-handler-content-attributes. So we should consult an HTML editor like @domenic to see if we should deviate from this precedent and add an event handler content attribute _specifically_ on HTMLFencedFrameElement and no other elements.

I feel like this might make it impossible for parent elements to listen to the fenced click event as it bubbles up past the fenced frame element, which is undesirable. Although I could be wrong (that is, maybe adding this specific attribute _only_ to fenced frame elements will allow bubbling event handlers to work fine after all).

Reply:
Yes, it's best to follow the precedent of all other events in HTML. You would update https://html.spec.whatwg.org/multipage/webappapis.html#event-handlers-on-elements,-document-objects,-and-window-objects and https://html.spec.whatwg.org/multipage/webappapis.html#globaleventhandlers

Reply:
Saw domenic's thumbs-up, so I went ahead and updated this sentence with the new wording.